rest/src/main/java/no/nav/vedtak/felles/prosesstask/rest/ProsessTaskRestTjeneste.java

```diff
@@ -71,7 +72,7 @@ public ProsessTaskRestTjeneste(ProsessTaskApplikasjonTjeneste prosessTaskApplika
             @ApiResponse(responseCode = "202", description = "Prosesstaskens oppdatert informasjon"),
             @ApiResponse(responseCode = "500", description = "Feilet pga ukjent feil eller tekniske/funksjonelle feil")
     })
-    @BeskyttetRessurs(action = CREATE, ressurs = DRIFT)
+    @BeskyttetRessurs(action = CREATE, resource = FORELDREPENGER_DRIFT, ressurs = DRIFT)
```

Comment:
bør være noe annet enn FORELDREPENGER_DRIFT her, trenger støtte K9 også

Reply:
Har opprettet tidligere en issue på dette #37. De fleste appene workarounder problemet ved å overskrive foreldrepenger sin tilgang med en egen i PdpRequestBuilderImpl men dette er ikke en bra løsning på sikt.

Reply:
Ja en konfig verdi ville være helt topp, problemet er at @BeskyttetRessurs forventer en konstant så det virker ikke helt som forventet.

Reply:
hmm, må enten lage adaptere eller noe form for indirecton  til abac attributt her

Reply:
lurer på om vi bare skulle gjort hele ProsessTaskRestTjeneste abstract og fjernet BeskyttetRessursAttributt, så må alle lage sin egen variant som delegerer?

Reply:
kan nå bruke property= for å refere et property herfra.  Må deretter sette i alle apper som tar i bruk